lib/iob/calculate.js

```diff
@@ -61,42 +61,50 @@ function iobCalcBiLinear(treatment, time, dia) {
 
 function iobCalc(treatment, time, dia, profile) {
 
-	var curve = profile.curve;
+    var curve = profile.curve;
 
-    if (curve === undefined || curve.toLowerCase() == 'bilinear') {
-    	return iobCalcBiLinear(treatment, time, dia);
+    if (curve === undefined ||  curve.toLowerCase() == 'bilinear') {
+        return iobCalcBiLinear(treatment, time, dia);
     };
-    
+
     curve = curve.toLowerCase();
-    
-    if (curve != 'novorapid' && curve != 'fiasp') {
-    	console.error('Unsupported curve function: "' + curve + '". Supported curves: bilinear, novorapid and fiasp.');
-    	return;
+
+    if (curve != 'fast-acting' && curve != 'ultrafast') {
```

Comment:
`rapid-acting` is the language commonly used in the USA for Novolog, Humalog, and Apidra, and `short-acting` for regular.  Would `rapid` and `ultra-rapid` work better here?